guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,9 @@
 // Do not update the puppet4 repo versions. They must stay at 6.3.
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+:RepoRHEL8ServerBaseos: rhel-8-for-x86_64-baseos-rpms
+:RepoRHEL8ServerAppstream: rhel-8-for-x86_64-appstream-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-6.11-for-rhel-8-x86_64-rpms
```

Comment:
While the others don't use a version macro either, they should. This prevents the inconsistency there is now (note 6.10 vs 6.11 here). In https://github.com/theforeman/foreman-documentation/pull/1054 we addressed that as well.